### PR TITLE
Upgraded table versions to allow for additional cell styles

### PR DESCRIPTION
**Pull request by ckoster22**
To test, run the style guide locally and validate the new Table example. You should expect to see cells that turn red when the number is less than 5.

image

### CODE REVIEW

File: src/Nri/Ui/Table/V5.elm [hunk 1]
```diff
@@ -0,0 +1,285 @@
+module Nri.Ui.Table.V5 exposing
+    ( Column, custom, string
+    , view, viewWithoutHeader
+    , viewLoading, viewLoadingWithoutHeader
+    )
+
+{-| Upgrading from V1:
```
Comment (brookeangel) [inline src/Nri/Ui/Table/V5.elm]: Should this have instructions for upgrading from V4?

File: styleguide-app/Examples/Table.elm [hunk 2]
```diff
@@ -33,12 +34,28 @@ example parentMessage state =
                 [ Table.string
                     { header = "First Name"
                     , value = .firstName
-                    , width = calc (pct 50) minus (px 125)
+                    , width = calc (pct 50) minus (px 250)
```
Comment (brookeangel) [inline styleguide-app/Examples/Table.elm]: Don't forget to update the name of this module in the config above!

File: styleguide-app/Examples/Table.elm [hunk 3]
```diff
@@ -33,12 +34,28 @@ example parentMessage state =
...
                     , value = .lastName
-                    , width = calc (pct 50) minus (px 125)
+                    , width = calc (pct 50) minus (px 250)
+                    , cellStyles = always []
+                    }
+                , Table.string
+                    { header = "# Submitted"
+                    , value = .submitted >> String.fromInt
+                    , width = px 125
+                    , cellStyles =
```
Comment (brookeangel) [inline styleguide-app/Examples/Table.elm]: 😻 yay thanks for showing this in the styleguide!

File: src/Nri/Ui/Table/V5.elm (hunk 1 above)
Comment (ckoster22) [reply src/Nri/Ui/Table/V5.elm]: Nice catch. Added

**Pull request merged**